### PR TITLE
Localize content/en/docs/reference/glossary/daemonset.md in hindi

### PR DESCRIPTION
**Pull request by taniaduggal**
Signed-off-by: Tania Duggal <103496926+taniaduggal@users.noreply.github.com>

fixes #36652

### CODE REVIEW

**Comment by taniaduggal** [comment]
@divya-mohan0209

**Comment by NitishKumar06** [comment]
> @divya-mohan0209

It is a request not to tag maintainers each time a PR has been raised. The reviewers have already been assigned automatically and would go through the changes once they are done with the other important work. Please be patient!

File: content/hi/docs/reference/glossary/daemonset.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+short_description: >
+  सुनिश्चित करें कि पॉड की प्रतिकृति क्लस्टर में नोड्स के एक सेट पर चल रही है।
+aka:
+tags:
+  - fundamental
+  - core-object
+  - workload
+---
+
+सुनिश्चित करता है {{<glossary_tooltip text="पॉड" term_id="pod" >}} की प्रतिकृति {{<glossary_tooltip text="क्लस्टर " term_id="cluster" >}} में नोड्स के एक सेट पर चल रही है।
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/daemonset.md:15]:
```suggestion
यह सुनिश्चित करता है कि {{< Glosary_tooltip text="क्लस्टर" term_id="cluster" >}} में नोड्स के सेट पर {{< Glosary_tooltip text="पॉड" term_id="pod" >}} की एक प्रतिलिपि चल रही है।
```

File: content/hi/docs/reference/glossary/daemonset.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+---
+title: डेमनसेट (DaemonSet)
+id: daemonset
+date: 2018-04-12
+full_link: /docs/concepts/workloads/controllers/daemonset
+short_description: >
+  सुनिश्चित करें कि पॉड की प्रतिकृति क्लस्टर में नोड्स के एक सेट पर चल रही है।
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/daemonset.md:7]:
```suggestion
 यह सुनिश्चित करता है कि क्लस्टर में नोड्स के सेट पर पॉड की एक प्रतिलिपि चल रही है।
```

File: content/hi/docs/reference/glossary/daemonset.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  - fundamental
+  - core-object
+  - workload
+---
+
+सुनिश्चित करता है {{<glossary_tooltip text="पॉड" term_id="pod" >}} की प्रतिकृति {{<glossary_tooltip text="क्लस्टर " term_id="cluster" >}} में नोड्स के एक सेट पर चल रही है।
+
+<!--more-->
+
+आमतौर पर सिस्टम डेमन जैसे लॉग कलेक्टर(log collectors) और मॉनिटरिंग एजेंट(monitoring agents) को तैनात करने के लिए उपयोग किया जाता है जो प्रत्येक{{< glossary_tooltip text="नोड" term_id="node">}} पर चलना चाहिए।
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/daemonset.md:19]: Rephrasing + slight grammatical nit.
```suggestion
लॉग कलेक्टर (log collectors) और निगरानी एजेंट (monitoring agents) जैसे सिस्टम डेमॉन को तैनात करने के लिए उपयोग किया जाता है जो आमतौर पर प्रत्येक नोड पर चलने चाहिए।
```

File: content/hi/docs/reference/glossary/daemonset.md (hunk 1 above)
Comment (Babapool) [reply content/hi/docs/reference/glossary/daemonset.md]: Shouldn't it be `glossary_tooltip` instead of  `Glosary_tooltip` ?

Comment (NitishKumar06) [reply content/hi/docs/reference/glossary/daemonset.md]: I'm also thinking about the same.

**Comment by NitishKumar06** [comment]
The checks are failing. Kindly look at the CI checks and feel free to unhold the PR once the checks pass.
/hold

File: content/hi/docs/reference/glossary/daemonset.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+short_description: >
+ यह सुनिश्चित करता है कि क्लस्टर में नोड्स के सेट पर पॉड की एक प्रतिलिपि चल रही है।
+aka:
+tags:
+  - fundamental
+  - core-object
+  - workload
+---
+
+यह सुनिश्चित करता है कि {{< Glosary_tooltip text="क्लस्टर" term_id="cluster" >}} में नोड्स के सेट पर {{< Glosary_tooltip text="पॉड" term_id="pod" >}} की एक प्रतिलिपि चल रही है।
```
Comment (Babapool) [inline content/hi/docs/reference/glossary/daemonset.md:15]: Now the Prow CI checks should not fail. 
```suggestion
यह सुनिश्चित करता है कि {{< glosary_tooltip text="क्लस्टर" term_id="cluster" >}} में नोड्स के सेट पर {{< glosary_tooltip text="पॉड" term_id="pod" >}} की एक प्रतिलिपि चल रही है।
```

**Comment by tengqm** [comment]
```
"content/hi/docs/reference/glossary/daemonset.md :15:63": failed to extract shortcode: template for shortcode "glosary_tooltip" not found
```

File: content/hi/docs/reference/glossary/daemonset.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+short_description: >
+ यह सुनिश्चित करता है कि क्लस्टर में नोड्स के सेट पर पॉड की एक प्रतिलिपि चल रही है।
+aka:
+tags:
+  - fundamental
+  - core-object
+  - workload
+---
+
+यह सुनिश्चित करता है कि {{< glosary_tooltip text="क्लस्टर" term_id="cluster" >}} में नोड्स के सेट पर {{< glosary_tooltip text="पॉड" term_id="pod" >}} की एक प्रतिलिपि चल रही है।
```
Comment (Babapool) [inline content/hi/docs/reference/glossary/daemonset.md:15]: Misspelt my suggestion of glossary while suggesting the change last time, apologies for that. Thank you @tengqm for pointing this out.
```suggestion
यह सुनिश्चित करता है कि {{< glossary_tooltip text="क्लस्टर" term_id="cluster" >}} में नोड्स के सेट पर {{< glossary_tooltip text="पॉड" term_id="pod" >}} की एक प्रतिलिपि चल रही है।
```

**Comment by kbhawkey** [comment]
@taniaduggal ,
I am closing this pull request.
When you have time to work on the changes again, feel free to reopen the PR.
/close

**Pull request closed without merging**